wolfcrypt/test/test.c

```diff
@@ -172,10 +172,16 @@ _Pragma("GCC diagnostic ignored \"-Wunused-function\"")
     #endif
     #include <android/log.h>
 
+    #ifndef ANDROID_v454 /* See fips/android/wolfCrypt_v454_android */
```

Comment:
I think this would be better structured as:

```
#ifdef ANDROID_V454
    #ifndef NO_FILESYSTEM 
    ...
#else
    ...
#endif
```
Since ANDROID_V454 seems like the special case here, not the generic fallback case.

Reply:
Done.